packages/entrykit/playground/wagmiConfig.ts

```diff
@@ -0,0 +1,40 @@
+import { Chain, http } from "viem";
+import { anvil } from "viem/chains";
+import { createWagmiConfig } from "../src/createWagmiConfig";
+import { chainId } from "./common";
+
+const chains = [
+  {
+    ...anvil,
+    rpcUrls: {
+      ...anvil.rpcUrls,
+      // TODO: automatically grant allowance in anvil instead of requiring the service
+      // quarryPassIssuer: {
+      //   http: ["http://127.0.0.1:3003/rpc"],
+      // },
```

Comment:
```suggestion
```
is this solved by using the generous paymaster (or anvil top up) locally?

Reply:
the entrykit playground is _our_ development environment, nothing shipped to users, so leaving this will let us play with quarry stuff locally

there tends to be a lot of things to configure in various spots, so commenting this out here is sort of a breadcrumb trail for myself to remember all the things to turn on before getting things wired up properly